control/tap-ctl-stats.c

```diff
@@ -105,11 +105,17 @@ tap_ctl_stats_fwrite(pid_t pid, int minor, FILE *stream)
 
 	prot  = PROT_READ|PROT_WRITE;
 	flags = MAP_ANONYMOUS|MAP_PRIVATE;
-	bufsz = sysconf(_SC_PAGE_SIZE);
+
+	err = sysconf(_SC_PAGE_SIZE);
+	if (err < 1) {
```

Comment:
From the man page:
PAGESIZE - _SC_PAGESIZE
Size of a page in bytes. Must not be less than 1. (Some systems use PAGE_SIZE instead.)